pkg/web/static/js/transactionAcceptPreview.js

```diff
@@ -12,6 +12,8 @@ document.body.addEventListener('htmx:afterSettle', () => {
   // Set the list of options in the network dropdown and display the value selected by the requester.
   const networkEl = document.getElementById('selected-network');
   const networkValue = networkEl?.value;
+  // Add placeholder in case the network value is empty.
+  networksArray.unshift({ 'placeholder': true, 'text': 'Select a network', 'value': '' });
```

Comment:
I attempted to handle this and other select options in the constants file, but then none of the options appeared. I assume it's because this is inside an `htmx:afterSettle` event.

Reply:
Mind adding a comment to reflect this? Just so we don't forget.